### ISSUE TITLE
android: Show message formatting options in compose text input menu bar.

### ISSUE THREAD

**Issue by jainkuniya** [zulip/zulip-mobile#3054]
https://cl.ly/d4ecff026e77

<img width="434" alt="image" src="https://user-images.githubusercontent.com/18511177/46912968-8140ec00-cfa1-11e8-8669-001b8ec3364b.png">

Eg: Like **bold**, and _italic_ menu options shown in the above image.

This will help in formatting message and improve message composing experience.

@zulipbot claim

Let's first focus on #2823, because it requires changes in the RN.

**Comment by jainkuniya** [comment]
BTW this can also be used in writing markdown 😄 
https://cl.ly/05a8ebf7998e

**Comment by borisyankov** [comment]
Yeah, but this not being such a key part of the app, and people that write bold/italics on the web app will just try using the `*`s to do the effect, I don't think it is that important (even if RN supported customizing the context menus by default)
